config/defaults.ini

```diff
@@ -466,6 +466,7 @@ apps_add_mandatory =
 [flatpak-remote-flathub]
 repo_file = https://dl.flathub.org/repo/flathub.flatpakrepo
 apps_add_mandatory =
+  com.vixalien.decibels
```

Comment:
Yes, on second thoughts, your line of reasoning makes sense: being able to play audio files is essential functionality, so decibels should go here; while having a music library app is not.